frontend/src/lib/constants.tsx

```diff
@@ -80,8 +80,6 @@ export const FEATURE_FLAGS = {
     NEW_SESSIONS_PLAYER: 'new-sessions-player', // owner: @rcmarron
     NEW_SESSIONS_PLAYER_EVENTS_LIST: 'new-sessions-player-events-list', // owner: @rcmarron / @alexkim205
     BREAKDOWN_BY_MULTIPLE_PROPERTIES: '938-breakdown-by-multiple-properties', // owner: @pauldambra
-    GROUP_ANALYTICS: 'group-analytics', // owner: @macobo
-    GROUP_ANALYTICS_INTRODUCTION: 'group-analytics-introduction', // owner: @macobo
```

Comment:
I would keep this FF (upsell) instead and use the persisted flags, we may want to do adjustments on Cloud.

Reply:
We discussed this with Marcus, didn't want to keep noisy flags around.